docker-compose.yml

```diff
@@ -24,6 +24,8 @@ services:
       - ETH_RPC_HOST=http://ganache
       - ETH_RPC_PORT=8545
       - ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
+      - LOG_LEVEL=debug
```

Comment:
ditto re: 'important' as the default

Reply:
Actually if this is in dev mode than i think debug makes sense here

Reply:
Ah, I didn't realize that this is specifically specifying a 'dev' config.  In that case leaving as debug sgtm